<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nightlies</class>
 <widget class="QWidget" name="Nightlies">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Last known good date</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>First known bad date</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="toolTip">
        <string>you can choose a specific repository name here</string>
       </property>
       <property name="text">
        <string>Repository</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="repository">
       <property name="toolTip">
        <string>repository like mozilla-aurora|mozilla-beta|...</string>
       </property>
       <property name="placeholderText">
        <string>you can choose a specific repository name here</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="NightlyInputSelection" name="start_date" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="NightlyInputSelection" name="end_date" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NightlyInputSelection</class>
   <extends>QWidget</extends>
   <header>mozregui.utils</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
